datasets/vergunningen/vergunningen.json

```diff
@@ -0,0 +1,87 @@
+{
+  "type": "dataset",
+  "id": "vergunningen",
+  "title": "Vergunningen",
+  "status": "beschikbaar",
+  "description": "Vergunningen",
+  "version": "0.0.1",
+  "crs": "EPSG:28992",
+  "tables": [
+    {
+      "id": "bedandbreakfast",
+      "title": "Bed & Breakfastvergunning beschikbaarheidsindicatie",
+      "description": "Indicatie beschikbaarheid voor een Bed & Breakfastvergunning per wijk. Per wijk is een maximaal aantal vergunningen beschikbaar.",
+      "type": "table",
+      "schema": {
+        "$schema": "http://json-schema.org/draft-07/schema#",
+        "type": "object",
+        "additionalProperties": false,
+        "required": ["schema", "id"],
+        "display": "id",
+        "properties": {
+          "schema": {
+            "$ref": "https://schemas.data.amsterdam.nl/schema@v1.1.1#/definitions/schema"
+          },
+          "id": {
+            "type": "integer",
+            "description": "Unieke aanduiding van een wijk met een beschikbaarheidindicatie voor een Bed & Breakfast vergunning."
+          },
+          "wijkNaam": {
+            "type": "string",
+            "description": "Naam van de wijk waar de beschikbaarheidindicatie Bed & Breakfast van toepassing is.",
+            "provenance": "wijk"
+          },
+          "categorieKleur": {
+            "type": "string",
+            "enum": ["green", "orange", "red"],
+            "description": "Kleurnaam beschikbaarheidindicatie Bed & Breakfast vergunning.",
+            "provenance": "availability_color"
+          },
+          "geometrie": {
```

Comment:
Klein dingetje. Het tabel attribuut "mainGeometry" is default "geometry". Omdat het hier nederlands gespeld is, moet er een expliciet "mainGeometry": "geometrie" worden toegevoegd.

Reply:
Goed om te weten, ik pas het aan